backend/burning-okr/burning-okr-logic/src/main/java/org/burningokr/service/okrUnit/OkrUnitServiceFactory.java

```diff
@@ -62,7 +62,7 @@ public OkrUnitServiceFactory(
     }
   }
 
-  public OkrUnitService<T> getUserService() {
+  public OkrUnitService<T> getUserOkrUnitService() {
```

Comment:
Warum nicht getOkrUnitService?

Reply:
Weil der `this.userService` in der Methode zurückgegeben wird und der vom Typ OkrUnitSerivceUsers ist `private OkrUnitServiceUsers<T> userService; ` André meinte getUserOkrUnitService hört sich besser an als getOkrUnitServiceUsers

Reply:
Alles klar, für mich war unkalr, woher das User kommt. Hatte mir allerdings nicht die Methode angeschaut